.github/workflows/test.yaml

```diff
@@ -38,10 +39,64 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: none # workaround for https://github.com/rhysd/actionlint/issues/170
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Save Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Pull a small test Docker image.
+        run: docker pull ghcr.io/linuxserver/baseimage-alpine:edge-version-f502fcfd
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: write
```

Comment:
Recommend adding `# for cache deletion` so that we know when this permission might need to be removed in the future.